### PR TITLE
update tx page

### PR DESCRIPTION
**Pull request by hackmod**
update tx page by @kimmyeonghun 
- [x] show tx status
- [x] etherscan.io style fix (hash => TxHash, blockNumber => Block Height, etc)
- [x] convert `gasPrice` wei unit to ether
- [x] more human readable numbers (by me)
- [x] show Gwei unit `GasPrice` (me)

image

### CODE REVIEW

File: routes/web3relay.js [hunk 1]
```diff
@@ -81,11 +81,37 @@ exports.data = function(req, res){
...
           if (!err && block)
             ttx.timestamp = block.timestamp;
-          ttx.isTrace = (ttx.input != "0x");
+          ttx.isTrace = false;
+          if (ttx.input != "0x") {
+            // check if input data is a string or contact data
+            if (ttx.to != null) {
+              var bytecode = web3.eth.getCode(ttx.to);
+              if (bytecode == "0x") {
+                // decode string data
```
Comment (naikmyeong) [inline routes/web3relay.js]: Can you restyle this section like etherscan?

default

**Comment by hackmod** [comment]
rebased

**Comment by btenterprise2020** [comment]
Feature replaced with https://github.com/ethereumclassic/explorer/pull/288

**Comment by btenterprise2020** [comment]
It is bad to fetch data from web3......

**Comment by btenterprise2020** [comment]
Rebase your PR!

**Pull request closed without merging**